src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +24,29 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public static final int SYMBOLS_NUMBER = 5;
+
+    public String getUniqueCharacters(String fileName) throws FileNotFoundException {
+        Set<Character> set = new TreeSet<>();
+        try (FileInputStream fin = new FileInputStream(fileName)) {
+            int symbol = 0;
+            while ((symbol = fin.read()) != -1) {
+                if (Character.isLetter((char) symbol)) {
+                    set.add(Character.toLowerCase((char) symbol));
+                }
+            }
+        } catch (FileNotFoundException e) {
+            throw new FileNotFoundException("File " + fileName + " does not exist");
+        } catch (IOException e) {
+            throw new RuntimeException("File " + fileName + " does not exist");
+        }
```

Comment:
let's remove these two `catch` blocks at all
if `try` block results in a FileNotFoundException - we might not catch it because we have `throws FileNotFoundException` statement in method signature

Reply:
but what if we have other IOException? without this construction it couldn`t pass tests because there are one where my method required to throw FileNotFound. If I have only IOException catching, it catches FileNotFound too